docker/unified-builder

```diff
@@ -38,6 +38,29 @@ ENV VERSION=AUTO_STRICT
 
 ENV PATH=$PATH:/root/.cargo/bin
 
+# Download OPA binary
+ARG OPA_VERSION=0.49.0
+RUN uname_S="$(uname -s)" && \
+  uname_M="$(uname -m)" && \
+  if [ "$uname_S" = "Linux" ]; then \
+  OS=linux && \
+  OPA_SUFFIX=_static; \
+  elif [ "$uname_S" = "Darwin" ]; then \
+  OS=darwin; \
+  fi && \
+  if [ "$uname_M" = "x86_64" ]; then \
+  ARCH=amd64; \
+  elif [ "$uname_M" = "arm" ]; then \
+  ARCH=arm64 && \
+  OPA_SUFFIX=_static; \
+  elif [ "$uname_M" = "arm64" ] || [ "$uname_M" = "aarch64" ]; then \
```

Comment:
Rather than an `if` just do a substitution  along the lines of 
```
uname -m | sed -e 's/x86_64/amd64/' -e 's/aarch64/arm64/'
```
Not sure you even need the `uname -m` since there is a docker built in var for TARGETARCH